### DISCUSSION TITLE
Jagged edges after geometry creation

### DISCUSSION THREAD

**Discussion by f-pucci** [pyvista/pyvista#5347] [Q&A]
Hello everyone, 
I am experiencing some problems with some geometries I generate with PyVista.
From an unstructured grid, to which I assigned some values, I generate my geometries using the clip_scalar function.

When I then import my geometry to my mesher (e.g., cfMesh for openfoam, or remesh in Blender), some edges of my geometry appear jagged/stair-stepped, no matter which resolution I set:

![image](https://github.com/pyvista/pyvista/assets/126581990/aee84f85-c635-4fcc-a6a7-2a0acbda0a26)

However, if I plot the geometry after I generate it with pyvista, these edges do not appear. I am afraid that this is due to the very acute angle that I have in correspondence of the edge. 

![image](https://github.com/pyvista/pyvista/assets/126581990/eab9cb88-6235-4a73-9a5c-5e023b83026e)

Can anyone suggest a solution to this problem? Thank you so much in advance!

**Comment by banesullivan** [comment]
I'm not too sure what's going on here. Would you please try to demonstrate what you are doing with a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)?

**Reply by f-pucci** [reply]
The geometry generation process goes quite smoothly and I have no problem with it so far.
Once I import the geometry into blender, and try to remesh it, I get the problem of the jagged edges after the remesh process.

I believe it's a curvature problem, because if I apply a "Curvature flipping optimization" filter in meshlab, elements in correspondence of those jagged edges disappear.

The stl itself does not present those, but then when I try to mesh it with OpenFOAM/cfMesh or with blender, I get this problem.
With OpenFOAM/snappyHexMesh I don't get that problem.

**Reply by MatthewFlamm** [reply]
It sounds like there are several moving parts here.  The problem is not observed in PyVista but in Blender _after_ remeshing in Blender.  This sounds more like a Blender problem, not a PyVista problem.

**Reply by f-pucci** [reply]
The problem also shows up in other meshers, cfMesh from OpenFoam or MeshLab.

**Reply by MatthewFlamm** [reply]
And the problem sounds like it is only viewing in Blender or PyVista?

**Reply by f-pucci** [reply]
In blender, meshlab and also openfoam/cfMesh after using any of their remeshing algorithms

**Reply by banesullivan** [reply]
So this is more of a general re-meshing topic/question regardless of pyvista then? 

I'm really confused by this and don't know how to help without some code to reproduce the phenomena myself
